### ISSUE TITLE
Violet Hold

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5739]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
If the door closes when the portals start then anyone who's behind door will be locked there since the npc to teleport you inside is missing. This issue is often a problem when someone is starting the instance from the NPC and someone is still not inside, or when someone is leaving and we get a new player he is stuck after the door cuz there's no npc to talk with to teleport inside. https://www.wowhead.com/npc=30658/lieutenant-sinclari#comments
**Expected behaviour**: Tell us what should happen instead
Lieutenant Sinclari should be behind the closed doors so ppl can talk to her and teleport inside
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by RandomXD** [comment]
there's a new bug at zuramat too, after he dies his nether thingy still throws shadow bolts at the group every few secs untill his corpse despawns http://prntscr.com/o25pbs

**Comment by talkaboom** [comment]
> there's a new bug at zuramat too, after he dies his nether thingy still throws shadow bolts at the group every few secs untill his corpse despawns http://prntscr.com/o25pbs

Not really new. It happened to me a few times, some over a year ago. Sometimes it stops firing before the next boss, sometimes it stays till the end.
Seems to trigger if the boss dies really fast or while casting the shadow binding spell.

**Comment by Aridam** [comment]
Bump -- NPC still missing at VH. Ended up wiping due to this.
https://imgur.com/a/Jkp5rBi
